package.json

```diff
@@ -11,14 +11,18 @@
     "lint": "npm run typecheck && next lint --dir . --ignore-path .gitignore",
     "lint_fix": "npm run typecheck && next lint --fix --dir . --ignore-path .gitignore",
     "lint:ci": "next lint --dir . --ignore-path .gitignore && npm run typecheck",
-    "test": "start-server-and-test start:test_server 3200 cypress:all",
-    "test:component": "start-server-and-test start:test_server 3200 cypress:component",
-    "test:e2e": "start-server-and-test start:test_server 3200 cypress:e2e",
+    "test": "npm run cypress:all && npm run jest:all",
+    "test:component": "npm run cypress:component && npm run jest:component",
+    "test:e2e": "cypress:e2e",
     "test:coverage": "npm run coverage:generate && npm run coverage:run_tests || npm run coverage:report",
     "cypress:all": "npm run cypress:component && npm run cypress:e2e",
-    "cypress:component": "npx cypress run --component",
-    "cypress:e2e": "npx cypress run --e2e",
+    "cypress:component": "start-server-and-test start:test_server 3200 _cypress:component",
+    "_cypress:component": "npx cypress run --component",
+    "cypress:e2e": "start-server-and-test start:test_server 3200 _cypress:e2e",
+    "_cypress:e2e": "npx cypress run --e2e",
```

Comment:
I remembered the other syntax you can use for this. `start-server-and-test` accepts non-npm commands if you include them in single quotes, so instead of:
```
    "cypress:component": "start-server-and-test start:test_server 3200 _cypress:component",
    "_cypress:component": "npx cypress run --component",
```
we can write
```
    "cypress:component": "start-server-and-test start:test_server 3200 'npx cypress run --component'",
```
and avoid the redundant commands. Sorry for not thinking of this when we were talking about it!